_posts/2024-05-16-introduction-to-web-neural-network-api.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: 'Introduction to Web Neural Network API (WebNN)'
+date: 2024-05-16 14:55:22 +0100
+author: Paul Cooper
+avatar: https://avatars.githubusercontent.com/u/722868?s=200&v=4
+categories: get-started
+---
+
+## Introduction to Web Neural Network API (WebNN)
```

Comment:
Please remove line 9 and 10 including `## Introduction to Web Neural Network API (WebNN)`.

The blog template uses "title" value of the boilerplate directly.